CONTRIBUTING.md

```diff
@@ -242,6 +242,10 @@ who you are:
 $ git config --global user.name "J. Random User"
 $ git config --global user.email "j.random.user@example.com"
 ```
+Please make sure this local email is also added to your
+[GitHub email list](https://github.com/settings/emails) so that your commits
+will be properly associated with your account and you will be timely promoted
```

Comment:
Maybe remove `timely`, it seems a bit out of place here.

I _think_ timely is actually an adjective, not an adverb (English - nonsense as usual).

Reply:
Done.